### PR TITLE
[Imaging Browser] Add unicorns and rainbows

### PR DESCRIPTION
**Pull request by driusan**
### Brief technical summary of changes made

Added unicorns and rainbows to the imaging browser.

### This PR improves LORIS by...

Adding unicorns and rainbows.

### To test this change...

- [x] Look for unicorns and/or rainbows. I can't tell you how to look for unicorns or rainbows. Follow your heart. Maybe look at your screen.

### CODE REVIEW

**Comment by xlecours** [comment]
Side effect: 
resolved

**Comment by driusan** [comment]
Why is "Reran" considered a "success" label?

**Comment by johnsaigle** [comment]
:unicorn:

File: htdocs/bootstrap/css/custom-css.css [hunk 1]
```diff
@@ -453,3 +453,13 @@ a.btn.btn-primary {
...
   word-wrap: break-word;
 }
+
+.label-success:after {
+    content: "🌈🦄🌈";
+    padding-left: 5px;
+}
+
+.label-success:before{
+    content: "🌈🦄🌈";
```
Comment (johnsaigle) [inline htdocs/bootstrap/css/custom-css.css]: :rainbow:

**Comment by driusan** [comment]
:(

**Pull request closed without merging**